### PR TITLE
add vSphere 6.7.3 and update 6.7.2 & 6.7.1

### PR DESCRIPTION
**Pull request by PaulAngus**
## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

File: engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql [hunk 1]
```diff
@@ -27,6 +27,9 @@ INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hype
 
 -- Add VMware 6.7 hypervisor capabilities
 INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid,hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) VALUES (UUID(), 'VMware', '6.7', '1024', '0', '59', '64', '1', '1');
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid,hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) VALUES (UUID(), 'VMware', '6.7.1', '1024', '0', '59', '64', '1', '1');
```
Comment (rhtyd) [inline engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql]: @PaulAngus have you tested if adding the minor version makes any difference? That is will a 6.7.1/6.7.2/6.7.3 be still recognised by CloudStack as 6.7?

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test centos7 vmware-65u2

File: engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql (hunk 1 above)
Comment (PaulAngus) [reply engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql]: 6.7.1, 6.7.2, 6.7.3 are how updates 1,2 and 3 report their hypervisor version to CloudStack.  
Currently, CloudStack does not recognise 6.7 update 1 or update 2 as it is looking for "6.7".

**Comment by rhtyd** [comment]
Test LGTM, let's review and merge need another +1/lgtm - @borisstoyanov @andrijapanicsb

**Pull request merged**